package.json

```diff
@@ -49,14 +49,11 @@
     "eslint": "^6.7.2",
     "eslint-plugin-spellcheck": "0.0.17",
     "graphql": "^15.5.0",
-    "highlight.js": "~9.14.0",
```

Comment:
This fixes https://github.com/advisories/GHSA-7wwv-vh3v-89cq, so @ehelms will be happy :)